### DISCUSSION TITLE
Highways

### DISCUSSION THREAD

**Discussion by singetail246** [voxeltycoon/issues#743] [General]
The idea of highways is to have a multi-lane road. No buildings or off roads will be built alongside it. You could specify the lane you wish the vehicle to prioritize and perhaps vehicles even get a little boost to their speed when on the highways. They could be used to help out with longer distance transport where trains might not be feasible.

**Comment by victorleblais** [comment]
There is a rework of road and rails in the works 
![image](https://user-images.githubusercontent.com/4365075/118280015-13982c00-b4cc-11eb-97bc-1c43c394c042.png)
https://voxeltycoon.xyz/roadmap
https://voxeltycoon.xyz/devlog/ea-faq/

It will allow to have more road types, some may come from the game itself, but I'm sure Mod Makers will make a lot more road types !
